src/services/severa-api-service.ts

```diff
@@ -27,6 +27,11 @@ export interface SeveraApiService {
   getPreviousWeekHours: (severaUserId: string) => Promise<SeveraResponsePreviousWorkHours[]>;
   getFilteredWorkHoursForUsers: (users: { guid: string }[], severaProjectId: string, startDate?: string, endDate?: string) => Promise<SeveraResponseWorkHours[]>;
   getWorkHoursForUser: (severaUserId: string, startDate?: string, endDate?: string) => Promise<SeveraResponseWorkHours[]>;
+  getResourceAllocationsByUserOrAll: (severaUserId: string | undefined, optInUsers: { guid: string }[]) => Promise<SeveraResponseResourceAllocation[]>;
+  checkKeywordExists: (keyword: string) => Promise<any>;
+  fetchUserByEmail: (email: string) => Promise<any>;
+  getUserKeywords: (userGuid: string) => Promise<any>;
+  updateSeveraOptInKeyword: (userGuid: string, isSeveraOptIn: string, isSeveraOptInKeywordGuid: string) => Promise<any>;
```

Comment:
Lets avoid typing the return as any, we should be able to get the response type from the severa client, which it looks like is already set up in this project. There is also the use of SeveraResponse types e.g. SeveraResponseWorkhours which are custom types created by us, but again these could be replaced by the types generated from the client.

Considering this I think it would be reasonable to make sure the severa API spec is generated (ideally just the models rather than the Apis) and then this could also be committed as part of this PR.